packages/css/.storybook/preview.js

```diff
@@ -13,7 +13,7 @@ export const parameters = {
     values: [
       {
         name: 'grey',
-        value: '#F8F8F8',
+        value: '#E5E5E5',
```

Comment:
Shouldn't we have the same change for the React project storybook/preview ?

Reply:
Yep.